hs_core/tests/api/rest/test_user_info.py

```diff
@@ -0,0 +1,44 @@
+import json
+
+from django.contrib.auth.models import Group
+
+from rest_framework.test import APIClient
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from hs_core.hydroshare import users
+
+
+class TestUserInfo(APITestCase):
+
+    def setUp(self):
+        self.maxDiff = None
+        self.client = APIClient()
+
+        self.group, _ = Group.objects.get_or_create(name='Hydroshare Author')
+        # create a user
+        self.email = 'test_user@email.com'
+        self.username = 'testuser'
+        self.first_name = 'some_first_name'
+        self.last_name = 'some_last_name'
+
+        self.user = users.create_account(
+            self.email,
+            username=self.username,
+            first_name=self.first_name,
+            last_name=self.last_name,
+            superuser=False)
+
+        self.client.force_authenticate(user=self.user)
+
+    def tearDown(self):
+        self.user.delete()
```

Comment:
@ironfroggy Is the tearDown section necessary? I remember you told me django test framework will take care of all clean-up. But looks here it is using "from rest_framework.test import APITestCase". Thanks

@selimnairb I would give a +1 once the above minor issue is settled. thanks